ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/CoverageTableViewToggle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+
+import { complianceEnhancedCoveragePath } from 'routePaths';
+
+export type CoverageTableViewToggleGroupProps = {
+    activeToggle: 'checks' | 'clusters';
+};
+
+function CoverageTableViewToggleGroup({ activeToggle }: CoverageTableViewToggleGroupProps) {
```

Comment:
To prevent ambiguity in the future, what do you think `Profile` instead of `Coverage` in the name?

What are pro and con to include or omit `View` in the name?

Reply:
I was originally imagining this component could be used for benchmark -> controls | clusters but now that I think about it, it won't be that easy. So `Profile` makes more sense. Will update